src/main/java/io/kestra/plugin/notifications/zenduty/ZendutyTemplate.java

```diff
@@ -38,40 +39,34 @@ public abstract class ZendutyTemplate extends ZendutyAlert {
     protected Map<String, Object> templateRenderMap;
 
     @Schema(
-        title = "Incident object's title"
+        title = "Incident title"
     )
     @PluginProperty(dynamic = true)
-    protected String title;
+    protected String message;
 
     @Schema(
-        title = "Incident object's status. 1 is triggered, 2 is acknowledged and 3 is resolved. Default value is 1"
-    )
-    @PluginProperty
-    protected Integer status;
-
-    @Schema(
-        title = "User object's username"
+        title = "Incident summary"
     )
     @PluginProperty(dynamic = true)
-    protected String assignedTo;
+    protected String summary;
 
     @Schema(
-        title = "Escalation Policy object's unique_id"
+        title = "Incident type"
     )
     @PluginProperty(dynamic = true)
```

Comment:
```suggestion
    @PluginProperty
```

Only strings can be rendered (also remove the runContext.render above)